Use "text/plain" content-type

jessfraz opened the pull request:
`gofmt`'d @exarkun commits from #9955 

Closes #9955 
Fixes #9954

exarkun committed:
Use the official mime type that exists instead of an imaginary one that does not.

Signed-off-by: Jean-Paul Calderone <exarkun@twistedmatrix.com>

Docker-DCO-1.1-Signed-off-by: Jessica Frazelle <jess@linux.com> (github: jfrazelle)

exarkun committed:
Change some instances of this mistake in the documentation as well.

Signed-off-by: Jean-Paul Calderone <exarkun@twistedmatrix.com>

Docker-DCO-1.1-Signed-off-by: Jessica Frazelle <jess@linux.com> (github: jfrazelle)

SvenDowideit commented:
Docs still LGTM - @fredlf @jamtur01

jamtur01 commented:
Docs LGTM

icecrime commented:
LGTM

crosbymichael commented on api/client/utils.go:
```diff
@@ -260,7 +260,7 @@ func (cli *DockerCli) monitorTtySize(id string, isExec bool) error {
 	gosignal.Notify(sigchan, signal.SIGWINCH)
 	go func() {
-		for _ = range sigchan {
+		for range sigchan {
```
This is not valid for Go < 1.4.  Maybe we could keep this the same as gccgo cannot even do the normal stuff let along this.

duglin replied:
I'm pretty sure I can ran into this too during some other PR. The latest gofmt wants this new format and drone fails because of it.

jfrazelle replied:
we fixed drone :D, i will fix the commit

jfrazelle commented:
updated, I added a commit bc those other weren't mine idk if I should have just updated their commit

jessfraz committed:
Fix range for go 1.3

Docker-DCO-1.1-Signed-off-by: Jessica Frazelle <jess@docker.com> (github: jfrazelle)

jfrazelle commented:
ummm drone?

crosbymichael commented:
Drone is tired, too much work for a single node

jfrazelle commented:
https://ci.dockerproject.com/github.com/docker/docker/commit/cd5902fc7bb0da36fedb6ff6a1bb8ef200b212d9?branch=9954-text-plain-content-type its in the queue :D

crosbymichael commented:
LGTM
